examples/sftp-multiple-keys/.header.md

```diff
@@ -0,0 +1,22 @@
+# SFTP Server with Multiple SSH Keys Example
+
+This example demonstrates how to create an AWS Transfer Family SFTP server with users that have multiple SSH public keys. This configuration showcases:
+
+- **Single Key User**: Traditional single SSH key configuration for backward compatibility.
+- **Multiple Key User**: User with multiple SSH keys for key rotation and redundancy.
+- **Key Rotation User**: User with both current and new keys for seamless rotation.
+
+## Key Features
+
+- Support for both single and multiple SSH keys per user
+- Demonstrates key rotation scenarios
+- Shows mixed user configurations in the same deployment
+- Uses service-managed identity provider
+- Includes S3 bucket with KMS encryption
+- CloudWatch logging enabled
+
+## Usage
+
+Replace the example SSH keys in the `users` local variable with your actual public keys before deploying.
```

Comment:
we should ideally not have any example SSH keys stored in the module. Load them up with tfvars for testing

Reply:
I don't think you have hard coded keys in the example. This can be rephrased